.goreleaser.yml

```diff
@@ -23,7 +23,7 @@ builds:
       - -trimpath
       - -v
     ldflags:
-      - main.version={{.Version}} -X main.commit={{.Commit}} -X main.date={{.Date}} -X main.builtBy=goreleaser`
```

Comment:
I am a bit confused by the backtick, I am assuming it was a typo? 😀

Reply:
It probably is :rofl: